library/kani/tests/ops.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use std::ops::{Bound, Range, RangeFrom, RangeInclusive, RangeTo, RangeToInclusive};
+
+fn _has_range_implementations() {
+    let _: Bound<u8> = kani::any();
+    let _: RangeFrom<u8> = kani::any();
+    let _: RangeInclusive<u8> = kani::any();
+    let _: RangeTo<u8> = kani::any();
+    let _: RangeToInclusive<u8> = kani::any();
```

Comment:
Thank you so much for this contribution. Could I ask one more thing before we merge it? Could you add one test case for each range implementation at [tests/kani/Arbitrary](https://github.com/model-checking/kani/tree/main/tests/kani/Arbitrary)? This not not only would tests these implementation in a Kani harness, but it would also serve as "usage documentation" later.